rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -198,7 +244,8 @@ def _get_signature(signature: Text, module: Any) -> NoReturn:
         """Retrieve a signature from a (hopefully loaded) TF model."""
         if not module:
             raise Exception(
-                "ConveRTFeaturizer needs a proper loaded tensorflow module when used. "
+                "ConveRTFeaturizerGraphComponent needs a proper loaded tensorflow "
```

Comment:
do we want to keep these like they are as we are gonna rename this back to `ConveRTFeaturizer` eventually once the old ones are removed?